packages/web-components/fast-components/src/skeleton/fixtures/base.html

```diff
@@ -0,0 +1,140 @@
+<DesignSystemProvider use-defaults>
+  <style>
+
+    h3 {
+      color: #ffffff;
+    }
+
+    .skeleton-container {
+      display: flex;
+      flex-direction: column;
+    }
+
+    .container {
+      display: flex;
+      flex-direction: column;
+    }
+
+    fluent-skeleton {
+      --neutral-fill-rest: #E1DFDD;
+      display: flex;
+      border-radius: 4px;
+      overflow: hidden;
+    }
+
+    .thing-1 {
+      width: 390px;
+      height: 250px;
+    }
+
+    .thing-3 {
+      width: 500px;
+      height: 250px;
+    }
+
+    svg {
+      position:absolute;
+      left: 0;
+      top: 0;
+    }
+
+    @keyframes shimmer {
+      0% {
+        transform: translateX(-100%);
+      }
+      100% {
+        transform: translateX(100%);
+      }
+    } 
+
+    .card-placeholder {
+      background-color: #fff;
+      padding: 20px;
+      width: 500px;
+      border-radius: 4px;
+    }
+    
+    .avatar-bone {
+      width: 50px;
+      height: 50px;
+    }
+
+    .text-bone {
+      margin-top: 10px;
+      height: 10px;
+    }
+
+    .button-bone {
+      width: 75px;
+      height: 30px;
+      margin-top: 20px;
+      margin-bottom: 10px;
+    }
+
+  </style>
+
+  <div class="skeleton-container">
+    <div class="container">
+      <div class="container">
+
+        <h3>Used as element blocks</h3>
+        <div class="card-placeholder">
+          <fluent-skeleton class="avatar-bone" shape="circle"></fluent-skeleton>
+          <fluent-skeleton class="text-bone" shape="rect"></fluent-skeleton>
+          <fluent-skeleton class="text-bone" shape="rect"></fluent-skeleton>
+          <fluent-skeleton class="text-bone" shape="rect"></fluent-skeleton>
+          <fluent-skeleton class="button-bone" shape="rect"></fluent-skeleton>
+        </div>
+  
+      </div>
+
+      <h3>Used as element blocks with shimmer effect element</h3>
+      <div class="card-placeholder">
+        <fluent-skeleton class="avatar-bone" shape="circle">
+          <span class="shimmer"></span>
```

Comment:
I wonder if this should be done as a boolean attr `shimmer` instead of a slotted child. Based on the boolean attr we could conditionally render the element in shadow dom, for example. You would have an API like this:

```html
 <fluent-skeleton shape="circle" shimmer> </fluent-skeleton>
```

Reply:
I like this, it makes sense. Will implement.